lib/widgets/picos_select.dart

```diff
@@ -60,49 +64,64 @@ class _PicosSelectState extends State<PicosSelect> {
   Widget build(BuildContext context) {
+    Color borderColor = Colors.grey.shade400;
     final BorderRadius borderRadius = BorderRadius.circular(7);
 
-    Color borderColor = Colors.grey.shade400;
+    OutlineInputBorder outlineInputBorder = OutlineInputBorder(
+      borderSide: BorderSide(
+        color: borderColor,
+      ),
+      borderRadius: borderRadius,
+    );
 
-    Border border = Border.all(color: borderColor);
+    OutlineInputBorder errorInputBorder = OutlineInputBorder(
+      borderSide: const BorderSide(
+        color: Colors.red,
+      ),
+      borderRadius: borderRadius,
+    );
 
     if (widget.disabled) {
       borderColor = Theme.of(context).disabledColor;
-      border = Border.all(
-        color: borderColor,
-        width: 0.35,
+      outlineInputBorder = OutlineInputBorder(
+        borderSide: BorderSide(
+          width: 0.35,
+          color: borderColor,
+        ),
+        borderRadius: borderRadius,
       );
     }
 
     return Padding(
       padding: const EdgeInsets.all(4),
       child: AbsorbPointer(
         absorbing: widget.disabled,
-        child: DecoratedBox(
-          decoration: BoxDecoration(
-            borderRadius: borderRadius,
-            border: border,
-          ),
-          child: DropdownButtonHideUnderline(
-            child: ButtonTheme(
-              alignedDropdown: true,
-              child: DropdownButton<String>(
-                borderRadius: borderRadius,
-                value: _dropdownValue,
-                iconSize: 30,
-                icon: const Icon(Icons.keyboard_arrow_down),
-                hint: Text(widget.hint ?? ''),
-                onChanged: (String? newValue) {
-                  setState(() {
-                    final String value = newValue ?? '';
-
-                    widget.callBackFunction(value);
-                    _dropdownValue = value;
-                  });
-                },
-                items: _createItemList(),
+        child: ButtonTheme(
+          alignedDropdown: true,
+          child: DropdownButtonFormField<String>(
+            decoration: InputDecoration(
+              contentPadding: const EdgeInsets.only(
+                right: 10,
               ),
+              enabledBorder: outlineInputBorder,
+              focusedBorder: outlineInputBorder,
+              errorBorder: errorInputBorder,
+              focusedErrorBorder: errorInputBorder,
             ),
+            borderRadius: borderRadius,
+            value: _dropdownValue,
+            icon: const Icon(Icons.keyboard_arrow_down),
+            hint: Text(widget.hint ?? ''),
+            onChanged: (String? newValue) {
+              setState(() {
+                final String value = newValue ?? '';
+
+                widget.callBackFunction(value);
+                _dropdownValue = value;
+              });
+            },
+            items: _createItemList(),
+            validator: widget.validator,
           ),
         ),
```

Comment:
```suggestion
        child: ButtonTheme(
          alignedDropdown: true,
          child: SizedBox(
            height: widget.validator == null ? null : 75,
            child: DropdownButtonFormField<String>(
              decoration: InputDecoration(
                contentPadding: const EdgeInsets.only(
                  right: 10,
                ),
                enabledBorder: outlineInputBorder,
                focusedBorder: outlineInputBorder,
                errorBorder: errorInputBorder,
                focusedErrorBorder: errorInputBorder,
              ),
              borderRadius: borderRadius,
              value: _dropdownValue,
              icon: const Icon(Icons.keyboard_arrow_down),
              hint: Text(widget.hint ?? ''),
              onChanged: (String? newValue) {
                setState(() {
                  final String value = newValue ?? '';

                  widget.callBackFunction(value);
                  _dropdownValue = value;
                });
              },
              items: _createItemList(),
              validator: widget.validator,
            ),
          ),
        ),
```
Small improvement. If validator is set the it makes automatically a small space for the error message, so the gaps between all imput elements look similar.